### DISCUSSION TITLE
outbound  bank transfer

### DISCUSSION THREAD

**Discussion by jester223** [DSJAS/DSJAS#8] [Ideas]
it might be a good idea to eventually create a outbound transfer with maybe a way to  capture info thats put as often scammers will put real account info that can be reported

**Comment by ejv2** [comment]
Great idea! I will add that to the current list of features that I am planning for a future update. Right now, we're working on the statistic system, which lets you track things like page hits, total money transferred and the largest amount "stolen" from your bank. You can see its progress if you download the latest alpha release (although I should warn you that it is a big work in progress).

Aside from the stats system, we're planning on making the bank's logo/icon customisable and adding the ability to upload custom assets through the admin dashboard. Another feature I had planned for a while is dynamic styles, that will let you keep your current theme enabled, but change certain elements of it (such as colors and fonts etc.).

Other stuff I would like to add before v1.0 is released is the ability to customise more arbitrary features of the site (such as emails, contact details and copyright messages). As, basically, the more moving parts we add that can be changed completely from site to site, the less likely it is that a DSJAS install will be spotted by a scammer.

As for outbound transfers, the current system isn't built for really managing any transfers that don't relate directly to accounts stored in the database, so I may have to add some moderately significant new stuff to manage this kind of thing. But this is absolutely something that I would be willing to write up and ship as a core feature of the site.
